cmd/import.go

```diff
@@ -47,7 +47,7 @@ func importRun(cmd *cobra.Command, args []string) error {
 
 	var toBeImported map[string]string
 
-	decoder := json.NewDecoder(in)
+	decoder := yaml.NewDecoder(in)
```

Comment:
I'm paranoid that this switch might not produce *precisely* the same result, but I'm struggling to come up with an actual example.